### PR TITLE
Add facundoalvarado9 mentor bio

### PR DESCRIPTION
**Pull request by FacundoAlvarado9**
New mentor here, answering the call for mentors! Java is my thing.

### CODE REVIEW

File: mentors/java/f.json [hunk 1]
```diff
@@ -7,4 +7,12 @@
       "avatar_url": null,
       "bio": "So Java would be my main language with something like 15+ years of experience. I don't particularly like it but it is what I know best!"
     }
```
Comment (rpottsoh) [inline mentors/java/f.json]: Line 9 needs a comma `,` after the `}`.

Comment (FacundoAlvarado9) [reply mentors/java/f.json]: Solved

**Comment by rpottsoh** [comment]
@ihid, Travis doesn't appear to be running.

**Comment by iHiD** [comment]
I don't know how to restart it. Sorry. I think merge if you are happy with it.

**Comment by rpottsoh** [comment]
@iHiD current repo settings will not allow me to merge this until all required checks have passed.

**Comment by rpottsoh** [comment]
@FacundoAlvarado9 Travis CI is not running against this PR for some reason.  I do not see a way at the moment to get it running.  I suggest just closing this PR and opening up another.

**Pull request closed without merging**